chalice/local/internal.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we can move this to a separate module, maybe `chalice.testclient` or `chalice.testutils`?

Reply:
Change the namespaces https://github.com/aws/chalice/pull/1193/commits/2c7ce41e7987ebd12e9ebde67300c64b1d1d0ad9